test/index.js

```diff
@@ -2160,12 +2161,16 @@ See https://github.com/sanctuary-js/sanctuary-def/tree/v${version}#ValidDate for
     eq (isRegexFlags ('gm'), true);
     eq (isRegexFlags ('im'), true);
     eq (isRegexFlags ('gim'), true);
+    eq (isRegexFlags ('dgimsuy'), true);
+    eq (isRegexFlags ('dgimsvy'), parseInt (process.versions.node, 10) >= 20);
```

Comment:
This shows that `'dgimsvy'` is a member of `RegexFlags` in Node.js v20.0.0 but not in earlier versions.